oauth2client/contrib/sql_alchemy.py

```diff
@@ -0,0 +1,86 @@
+"""OAuth 2.0 utilities for SQLAlchemy.
+Utilities for using OAuth 2.0 in conjunction with SQLAlchemy.
+Heavily inspired by equivalent Django module.
+"""
+
+from oauth2client.client import Storage as BaseStorage
+from sqlalchemy.types import PickleType
+
+
+__author__ = 'dominik@mdzn.pl (Dominik Miedzinski)'
```

Comment:
The `__author__` line. Git tracks all of this better than an attribute in a file can.